audioswitch/src/main/java/com/twilio/audioswitch/AudioDeviceChangeListener.kt

```diff
@@ -0,0 +1,11 @@
+package com.twilio.audioswitch
+
+/** A listener that receives the most recently available [AudioDevice]s.
+ * Receives a list of the most recently available [AudioDevice]s. Also provides the
+ * currently selected [AudioDevice] from [AudioDeviceSelector].
+ *
+ * @param audioDevices the list of [AudioDevice]s or null if none are available.
```

Comment:
Would empty list be better if none are available?

Reply:
I think null would be better here since null is part of the type system in Kotlin. If it were Java I'd say lets use an empty list.